internal/handlers/handlers.go

```diff
@@ -1 +1,68 @@
 package handlers
+
+import (
+    "github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+    "io"
+    "net/http"
+    "os"
+    "time"
+)
+
+func getForm(w http.ResponseWriter, r *http.Request) {
+    if r.Method != http.MethodGet {
+        http.Error(w, `{"error": "invalid method"}`, http.StatusMethodNotAllowed)
+        return
+    }
+
+    resp, err := os.ReadFile("index.html")
+    if err != nil {
+        http.Error(w, err.Error(), http.StatusInternalServerError)
+        return
+    }
+
+    w.Header().Set("Content-Type", "text/html; charset=utf-8")
+    w.Write(resp)
+}
+
+func postForm(w http.ResponseWriter, r *http.Request) {
+    err := r.ParseMultipartForm(1024)
+    if err != nil {
+        http.Error(w, `{"error": "can't parse"}`, http.StatusInternalServerError)
```

Comment:
Это проблема клиента
```suggestion
        http.Error(w, `{"error": "can't parse"}`, http.StatusBadRequest)
```